src/System.Diagnostics.DiagnosticSource/src/System/Diagnostics/HttpHandlerDiagnosticListener.cs

```diff
@@ -0,0 +1,370 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections;
+using System.Collections.Generic;
+using System.Net;
+using System.Reflection;
+using System.Reflection.Emit;
+
+// This HttpHandlerDiagnosticListener class is applicable only for .NET 4.5/4.6, and not for .NET core.
+// If you are making these changes, please test your changes manually via custom test applications.
```

Comment:
>We should be able to have the tests here. It is only that we don't run the automatically (so you have to run them by hand).

Doing this in the test folder will run the tests against netfx (Desktop):

"msbuild /t:rebuildandtest /p:TargetGroup=netfx"